api-reference/10 UI Components/GridBase/3 Methods/selectRows(keys_preserve).md

```diff
@@ -20,6 +20,10 @@ By default, this method call clears selection of previously selected rows. To ke
 
     <!--JavaScript-->widgetInstance.selectRows([5, 10, 12], true);
 
+If you specify DataGrid's key as composite, for example `key = ['id', 'name']`, you need call this method as follows:
```

Comment:
"For example" should be followed by a comma, but there would be too many commas in this sentence, so I suggest putting the example in brackets.

```suggestion
If you specify DataGrid's key as composite (for example, `key: ['id', 'name']`), you need to call this method as follows:
```